docs/config/hosting.md

```diff
@@ -0,0 +1,47 @@
+# Hosting
+
+Want to host Kowl under a sub-path (instead of `/`)? Then this is the document you're looking for.
+
+There are 3 config entries that should cover most scenarios.
+As you can see in [config/kowl.yaml](./kowl.yaml) the config entries can be found in the `server:` group.
```

Comment:
I guess this should be part of ## Config Entries?

Reply:
will fix